CONTRIBUTING.md

```diff
@@ -14,26 +14,27 @@ Found a security issue? Please disclose it responsibly. We have a team of develo
 
 ## Contributing to translations
 
-We use [WebTranslateIt][webtranslateit] to manage translation. Please read [our wiki page][translation-wiki] to know more about it. Except if your language isn't on webtranslateit, please do **not** open a pull request to update translations.
+We use [WebTranslateIt][webtranslateit] to manage translations of the app interface. Please read [our wiki page][translation-wiki] to find out more about this. If your language is on WebTranslateIt, please do **not** open a pull request to update translations. If you want to contribute to a language that is not featured on WebTranslateIt, please contact our developers to find out how to proceed.
```

Comment:
>If you want to contribute to a language that is not featured on WebTranslateIt, please contact our developers to find out how to proceed

Not too sure about that, I'd like to redirect to the wiki page, but we definitely need to update its content. @jhass you were the one who managed translations, what's the current recommendations to contribute to translations? Do you have some time to update https://wiki.diasporafoundation.org/Contribute_translations ?